inspirehep/modules/records/cli.py

```diff
@@ -0,0 +1,33 @@
+import click
+
+from .checkers import obtain_references_without_record, create_table_of_references_missing_record
+
+@click.group()
+def ref_wo_rec():
+	"""Commands to obtain references that have no record object"""
+
+
+@ref_wo_rec.command()
+def write_references():
+	click.echo(obtain_references_without_record)
+
+
+@ref_wo_rec.command()
+@ref_wo_rec.argument('file_name',type=str)
+@ref_wo_rec.option('--doi','-d',isflag=True, default=False,
+				   help='Write a file only with references with DOI')
+@ref_wo_rec.option('--arxiv','-a',isflag=True, default=False,
+				   help='Write a file only with references with arxiv id')
+def write_references_on_file():
+	reference_list = obtain_references_without_record
+	result_doi, result_arxiv = create_table_of_references_missing_record(data)
+
+	if doi:
+		with open('{0}_doi.txt'.format(file_name),'w') as f:
+			for item in result_doi:
+				f.write('{0}: {1}\n'.format(item[0].encode('utf-8'),str(item[1]).encode('utf-8')))
```

Comment:
look at the `click` documentation for file handling, I am sure it can handle this kind of things in a better way.